src/shared/test/trim-and-dedent-test.js

```diff
@@ -50,8 +52,7 @@ Goodbye, John!
 
         multiple
 lines
-with no indentation
-        `,
+with no indentation`,
```

Comment:
This test always felt wrong. The bar had an empty line at the end, and was itself the last line of the template string.

With these changes it now gets properly trimmed.